docs/codegen.md

```diff
@@ -44,21 +48,21 @@ To parse source code and produce ASTs locally:
 cabal new-repl lib:tree-sitter-python
 ```
 
-2. Set language extensions, `OverloadedStrings` and `TypeApplications`, and import relevant modules, `TreeSitter.Unmarshal`, `TreeSitter.Range` and `TreeSitter.Span`:
+2. Set language extensions, `OverloadedStrings` and `TypeApplications`, and import relevant modules, `TreeSitter.Unmarshal`, `Source.Range` and `Source.Span`:
 
 ```
 :seti -XOverloadedStrings
 :seti -XTypeApplications
 
-import TreeSitter.Span
-import TreeSitter.Range
+import Source.Span
+import Source.Range
 import TreeSitter.Unmarshal
```

Comment:
@aymannadeem what do you think about committing a `.ghci` file that does this?

Reply:
I have a `.ghci` file locally that takes care of this, happy to commit it since this is all pretty unusable without these key imports.